### DISCUSSION TITLE
No more pyxel module for Debian 12 ???

### DISCUSSION THREAD

**Discussion by cperrod** [kitao/pyxel#518] [Announcements]
Hi,

I can't find out a way to install Pyxel on Debian 12 + Python 3.11...no more python3-pyxel package...

What is the right way to install Pyxel ?

Thanls !

**Comment by DiddiLeija** [comment]
I don't know what's the installation method you're currently using, but I think you should be able to install Pyxel via `pip install pyxel`, or `python3 -m pip install pyxel` if you have several Pythons installed :)

**Comment by cperrod** [comment]
Thank you for your reply.

Sadly, Python packages in Debian 12 are now externally managed and have to
be installed with the system package manager, in which I can'd find any
python3-pyxel or so package...such a weird thing, I can't be the only one
complaining about the difficulty to install Pyxel in Debian 12, am I ???

Chris

Le ven. 2 févr. 2024 à 20:22, Diego Ramirez ***@***.***> a
écrit :

**Reply by chrzyki** [reply]
Hm, you should certainly be able to manage packages 'locally' rather than system-wide. If you've got `python3-venv` (`sudo apt install python3-venv`) available (and of course all of its dependencies) it should just be:

```
$ mkdir pyxel_project && cd pyxel_project
$ python3 -m venv venv
$ source venv/bin/activate
$ pip3 install pyxel
```

AFAIK there's nothing special about Debian (12) that prohibits you from doing that. Yes, Python packages that are available in the distribution's repositories can be installed from there, but you'll always have the ability to install in your own venv (or local environment, local path, etc.).

**Reply by cperrod** [reply]
Thank you for your reply.

Well, I was quite ignorant about virtual environments; I've installed pipx who now makes the job for me ;-)...

Regards
